src/components/organisms/table/alphabet-filter/alphabet-filter.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import { useEffect } from "react";
+import { TableConstants } from "@constants";
```

Comment:
I would create multiple constans files, as this shouldn't be part of TableConstants